pygmt/src/inset.py

```diff
@@ -0,0 +1,121 @@
+"""
+inset - Create inset figures.
+"""
+import contextlib
+
+import numpy as np
+from pygmt.clib import Session
+from pygmt.helpers import (
+    args_in_kwargs,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+
+@fmt_docstring
+@contextlib.contextmanager
+@use_alias(
+    D="location",
+    V="verbose",
+    F="border",
```

Comment:
We use `box` in other modules with similar syntax (e.g., [`logo`](https://pygmt-d8imhqura.vercel.app/api/generated/pygmt.Figure.logo.html), [`legend`](https://pygmt-d8imhqura.vercel.app/api/generated/pygmt.Figure.legend.html))